core/templates/pages/contributor-dashboard-page/services/translate-text-backend-api.service.ts

```diff
@@ -39,13 +49,32 @@ export class TranslateTextBackendApiService {
       return TranslatableTexts.createFromBackendDict(backendDict);
     });
   }
-
+  async blobtoBase64(blob: Blob): Promise<unknown> {
+    return new Promise<unknown> ((resolve, reject)=> {
+      const reader = new FileReader();
+      reader.onload = () => {
+        // Read the base64 data from result.
+        const dataurl = reader.result as string;
+        const prefix = /^data:image\/(gif|png|jpeg|svg\+xml|jpg);base64,/;
```

Comment:
Call this prefixRegex or similar, since it's not the literal prefix.

Reply:
Done.

Reply:
Done.